src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelProxyDiscoveryManager.cs

```diff
@@ -242,6 +242,7 @@ private void DiscoverTestsOnConcurrentManager(IProxyDiscoveryManager proxyDiscov
 
                 // Kick off another discovery task for the next source
                 var discoveryCriteria = new DiscoveryCriteria(new[] { nextSource }, this.actualDiscoveryCriteria.FrequencyOfDiscoveredTestsEvent, this.actualDiscoveryCriteria.DiscoveredTestEventTimeout, this.actualDiscoveryCriteria.RunSettings);
```

Comment:
>DiscoveryCriteria [](start = 44, length = 17)

check if overriden constructor for criteria

Reply:
Letting behavior consistent with TestRunCriteria and thus not adding overriden constructor accepting testCaseFilter arg